src/display/api.js

```diff
@@ -107,72 +108,72 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
- * @property {string}     [password] - For decrypting password-protected PDFs.
+ *   as cookies or authorization headers. The default is `false`.
+ * @property {string} [password] - For decrypting password-protected PDFs.
  * @property {TypedArray} [initialData] - A typed array with the first portion
  *   or all of the pdf data. Used by the extension since some data is already
  *   loaded before the switch to range requests.
- * @property {number}     [length] - The PDF file length. It's used for
- *   progress reports and range requests operations.
- * @property {PDFDataRangeTransport} [range]
- * @property {number}     [rangeChunkSize] - Specify maximum number of bytes
- *   fetched per range request. The default value is 2^16 = 65536.
- * @property {PDFWorker}  [worker] - The worker that will be used for
- *   the loading and parsing of the PDF data.
- * @property {number} [verbosity] - Controls the logging level; the
- *   constants from {VerbosityLevel} should be used.
- * @property {string} [docBaseUrl] - The base URL of the document,
- *   used when attempting to recover valid absolute URLs for annotations, and
- *   outline items, that (incorrectly) only specify relative URLs.
- * @property {string} [cMapUrl] - The URL where the predefined
- *   Adobe CMaps are located. Include trailing slash.
- * @property {boolean} [cMapPacked] - Specifies if the Adobe CMaps are
- *   binary packed.
- * @property {Object} [CMapReaderFactory] - The factory that will be
- *   used when reading built-in CMap files. Providing a custom factory is useful
- *   for environments without `XMLHttpRequest` support, such as e.g. Node.js.
- *   The default value is {DOMCMapReaderFactory}.
+ * @property {number} [length] - The PDF file length. It's used for progress
+ *   reports and range requests operations.
+ * @property {PDFDataRangeTransport} [range] - Allows for using a custom range
+ *   transport implementation.
+ * @property {number} [rangeChunkSize] - Specify maximum number of bytes fetched
+ *   per range request. The default value is 2^16 = 65536.
+ * @property {PDFWorker} [worker] - The worker that will be used for loading and
+ *   parsing the PDF data.
+ * @property {number} [verbosity] - Controls the logging level; the constants
+ *   from {@link VerbosityLevel} should be used.
+ * @property {string} [docBaseUrl] - The base URL of the document, used when
+ *   attempting to recover valid absolute URLs for annotations, and outline
+ *   items, that (incorrectly) only specify relative URLs.
+ * @property {string} [cMapUrl] - The URL where the predefined Adobe CMaps are
+ *   located. Include the trailing slash.
+ * @property {boolean} [cMapPacked] - Specifies if the Adobe CMaps are binary
+ *   packed or not.
+ * @property {Object} [CMapReaderFactory] - The factory that will be used when
+ *   reading built-in CMap files. Providing a custom factory is useful for
+ *   environments without `XMLHttpRequest` support, such as e.g. Node.js. The
```

Comment:
This comment is quite old, so you probably want to mention `Fetch API`/`XMLHttpRequest` here now.